tools/swarm/src/compose.rs

```diff
@@ -471,6 +477,20 @@ fn generate_key_pair(base_seed: Option<&[u8]>, additional_seed: &[u8]) -> KeyPai
     })
 }
 
+fn generate_hex_string_signature(chain_id: &ChainId, genesis_key_pair: &KeyPair) -> String {
+    let manifest_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let raw_block = iroha_genesis::RawGenesisBlock::from_path(
+        manifest_dir.join("../../configs/swarm/genesis.json"),
```

Comment:
`iroha_swarm` is not a tool to work only in our repo. CLI args should provide path to the genesis.

But, in this case, `iroha_swarm` will duplicate the functionality of `kagami genesis sign`. So, I think it would be OK if users provide the signature directly, i.e.:

```shell
iroha_swarm --genesis-signature 0022aafsdfew...
```